everware/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.9.0"
+version = "0.9.0"
```

Comment:
wait, here underscores for `__version__` is quite normal (it is the case for every package I've seen). I was complaining only about using underscores for function argument names. 
sorry for the confusion.